generators/client/templates/angular/src/main/webapp/app/admin/user-management/user-management-update.component.ts.ejs

```diff
@@ -71,6 +97,16 @@ export class UserMgmtUpdateComponent implements OnInit {
         }
     }
 
+    private updateUser(user): void {
+        user.login = this.editForm.get(['login']).value;
```

Comment:
if form and model structure matches, then, you can get model object directly using `this.editForm.value`

Reply:
do you mean that I could use user = this.editForm.value in this case? if it is, this is AWESOME!

Reply:
yes

Reply:
Yes if the model and the form are the same structure it works well :) good catch @vishal423

Reply:
Tried but it only works if all the fields are identical: unfortunately, there are case where fields are undefined so that it fails...

Reply:
@Tcharl I know this is probably old, but could you please specify which field where problematic, since I already implemented all of this in a blueprint. but maybe I missed something.